features/facebookImport/src/facebookImporter.jsx

```diff
@@ -143,10 +147,16 @@ const FacebookImporterApp = () => {
     return (
         <Router history={history}>
             <ImporterProvider>
-                <FileLoaderProvider parentContext={ImporterContext}>
+                <PolyImportProvider
+                    parentContext={ImporterContext}
+                    analyzeFile={analyzeFile}
```

Comment:
There's probably a more elegant way to do this, but I'm completely on board with keeping the changes minimal for now.